examples/cloudflare-workers-test/serverless.yml

```diff
@@ -4,7 +4,8 @@ components:
         myWorker:
```

Comment:
Let's make the `account_id` and `zone_id` placeholder values.

I'll make sure the documentation reflects how to find these